R/shinyform.R

```diff
@@ -84,16 +84,23 @@ loadData <- function(storage) {
 # Takes data from your shinyforms inputs and saves it to a flat file. 
 # Writes form inputs to a storage type and names it using a timestamp.
 # @param data Dataframe taken from input shiny object
-# @param storage A list with variable type defining users perferred type of storage and storage path
+# @param storage A list with variable type defining users perferred type of storage, filename and storage path
 saveDataFlatfile <- function(data, storage) {
-  fileName <- paste0(
-    paste(
-      format(Sys.time(), "%Y%m%d-%H%M%OS"),
-      digest::digest(data, algo = "md5"),
-      sep = "_"
-    ),
-    ".csv"
-  )
+  if(!is.null(formInfo$storage$filename)){
```

Comment:
This `saveDataFlatfile` function only has two parameters: data and storage. `formInfo` is not a parameter in this function. It can't be used here.

Reply:
Yeah sorry should have just been storage$filename